aksel.nav.no/website/pages/eksempler/switch/controlled-value.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { Switch } from "@navikt/ds-react";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+const Example = () => {
+  const [checkedValue, setCheckedValue] = useState("");
+
+  return (
+    <Switch
+      value="sms"
+      checked={checkedValue === "sms"}
+      onChange={(v) => setCheckedValue((x) => (x ? "" : v.target.value))}
```

Comment:
Småpirk, men folk er kanskje mer vant med `e` for event (i stedet for `v`). Kan eventuelt skrive `event` helt ut også.

Reply:
👌

Reply:
Done 🏁